torch/_inductor/codegen/xpu/device_op_overrides.py

```diff
@@ -30,5 +30,53 @@ def cpp_aoti_stream_guard(self):
     def cpp_getStreamFromExternal(self):
         return "at::xpu::getStreamFromExternal"
 
+    def kernel_header(self):
+        source_codes = """
+        #include <c10/core/DeviceGuard.h>
+        #include <c10/xpu/XPUStream.h>
+        #include <ATen/xpu/EmptyTensor.h>
```

Comment:
Do you have to directly include these c10/aten headers here?

Reply:
> Do you have to directly include these c10/aten headers here?

Hi, @desertfire: Yes, I flow the cuda implementation: https://github.com/pytorch/pytorch/blob/1c1d06a22c2534cca3c4c36f5e7947a67b0d1512/torch/_inductor/codegen/cuda/device_op_overrides.py#L35C1-L41C1

Reply:
But that is not used anywhere (likely only used in the previous non-ABI compatible mode). We need to limit direct access to libtorch in the generated code to avoid any accidental ABI BC breakage. We can only access through the C shim layer or through sanity-checked header-only files for some utility functions.

Reply:
Thanks @desertfire  I've removed those header. BTW could you please help review the PRs stacked above this PR, they are all created to support AOTI XPU, and ready for your review. Thanks.